### PR TITLE
Copyediting (mostly) on Aftershock's README.md

### PR DESCRIPTION
**Pull request by actual-nh**
#### Summary
None

#### Purpose of change

In the process of converting this into [a Wiki page](http://cddawiki.chezzo.com/cdda_wiki/index.php?title=Aftershock_README), I spotted a few things that I hadn't earlier (oops!).

#### Describe the solution

Edited (including to fit CDDA's style guide). Made the start clearer by swapping the two paragraphs.

#### Describe alternatives you've considered

Wincing every time I saw an error that I could have helped fix.

### CODE REVIEW

File: data/mods/Aftershock/README.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
+* Aftershock is being developed into a full conversion mod set in a mostly-abandoned extrasolar colony named Salus IV.  The player has arrived on Salus for various reasons (depending on their profession).  Eventually, we hope to have multiple potential end game situations, ranging from finding what you came here for and leaving the planet with enough money to retire, to more-secret endings.  There will no longer be zombies; instead, there are a variety of alien and human-based enemies.  Toe-to-toe combat will likely be more deadly in Aftershock, with more emphasis on picking your battles and only fighting when absolutely necessary.  Special locations may have a different combat aesthetic; for example, you may enter a dungeon-like zone with hordes of small weak enemies that can't be avoided.  As part of this concept, in which different situations require different solutions, we have implemented bionic slots but made the process of changing out CBMs much easier with less chance of failure.  This is to encourage players to change their loadout based on where they are going and what they are doing.
```
Comment (actual-nh) [inline data/mods/Aftershock/README.md:2]: Regarding "changing out CBMs" - is this already implemented?

File: data/mods/Aftershock/README.md [hunk 2]
```diff
@@ -54,7 +52,7 @@ Contains json data for vehicles, vehicle parts, and vehicle spawn groups.
 
 ### npcs
 
-Contains json data for npcs, factions, and design document for any new factions, specific new factions PrepPhyle and Whately Clan.  The PrepPhyle are a group of frontier squatters taking advantage of decaying colony infrastructure to set up an independent colonoy separate from Corporate and Earthgov influences.  The Whately Clan are a possibly-exiled branch of a corporate executive family.  They are mad, bad, and dangerous to know. Mutant NPCs are also located in this folder.
+Contains json data for npcs, factions, and design document for any new factions, specific new factions PrepPhyle and Whately Clan.  The PrepPhyle are a group of frontier squatters taking advantage of decaying colony infrastructure to set up an independent colony separate from Corporate and Earthgov influences.  The Whately Clan are a possibly-exiled branch of a corporate executive family.  They are mad, bad, and dangerous to know.  Mutant NPCs are also located in this folder.
```
Comment (actual-nh) [inline data/mods/Aftershock/README.md:55]:
```suggestion
Contains json data for npcs and factions, which are currently the new factions PrepPhyle and Whately Clan.  The PrepPhyle are a group of frontier squatters taking advantage of decaying colony infrastructure to set up an independent colony separate from Corporate and Earthgov influences.  The Whately Clan are a possibly-exiled branch of a corporate executive family.  They are mad, bad, and dangerous to know.  Mutant NPCs are also located in this folder.
```
I can't find a faction design document.

**Review by actual-nh — COMMENTED** [review]
A couple of questions for the maintainers.

File: data/mods/Aftershock/README.md (hunk 2 above)
Comment (John-Candlebury) [reply data/mods/Aftershock/README.md:55]: Its in-progress draft is in here: https://github.com/John-Candlebury/Cataclysm-DDA/wiki

File: data/mods/Aftershock/README.md (hunk 1 above)
Comment (John-Candlebury) [reply data/mods/Aftershock/README.md:2]: Yes, mostly as a modification of Autodoc skill multipliers

File: data/mods/Aftershock/README.md (hunk 2 above)
Comment (actual-nh) [reply data/mods/Aftershock/README.md:55]: Should I put in a link?

Comment (John-Candlebury) [reply data/mods/Aftershock/README.md:55]: Probably not yet.

**Review by John-Candlebury — APPROVED** [review]

**Pull request merged**